components/index.js

```diff
@@ -1 +1,18 @@
-// Add compents here
```

Comment:
you can ignore the index.js files for the most part. you are right -- each fiel should be its own react compentn